### ISSUE TITLE
Getting Params Error

### ISSUE THREAD

**Issue by rflores00** [mercadopago/sdk-android#23]
Hey there! I have throubles to test mercado pago SDK.. 

Steps:
1. I created a test user follow this official guide: [https://www.mercadopago.com.ar/developers/en/solutions/payments/basic-checkout/test/test-users/](url) 
2. I downloaded the example for android.. 
3. I replaced DUMMY_MERCHANT_PUBLIC_KEY and  DUMMY_MERCHANT_ACCESS_TOKEN fields with my own credentials from user that I created (step 1).

So, when I run the example apk into my phone, I put information from test card given from official documentation and put in field "Name and Last name" the prefix "APRO" to get approvation response. when I press Ok button, show me a loading page and after that I give a toast message with the text "params error".

Can you help me with this?

Best regards,

**Comment by rflores00** [comment]
Hey! Finally I did work my android app with all information and comments given by @scirielli in the other issue (https://github.com/mercadopago/sdk-android/issues/17i). 

Now I have some doubts, I'm getting approved transaction by doPayments method, but in the mercadopago,com control panel I don't see the operation, the payment did it from my android app is not charge to balance of test user. What is happen? I need to use real credit cards?

Regards

**Comment by scirielli** [comment]
Hi,
when you create a test user you have:
- the user name or email
- the user password
and you get:
- a public key
- a private client id and secret -> you create a private access token

Inside the method doPayments, if you create a payment using the test user's access token you should see it if you log in to MercadoPago site using the test user's username and password.

Is that what you're trying to do?

Regards,
SC

**Comment by rflores00** [comment]
Hi @scirielli, yes that is what I'm trying to do, when you talk about " you create access token".. I did not create anyone, I use the access token given by "https://www.mercadopago.com/mlv/account/credentials"

Regards,

**Comment by scirielli** [comment]
Indeed. Can you tell me a payment id that you're not seeing so I can understand what's going on?

**Comment by rflores00** [comment]
I'm using credit card visa sample of argentina, given by mercadoapago docs https://www.mercadopago.com.ar/developers/en/solutions/payments/custom-checkout/test-cards/

**Comment by scirielli** [comment]
What i'm asking is the id of the approved payment you get after calling the doPayment method. Ok?

**Comment by rflores00** [comment]
Oh yeah! sorry for my bad, this is one I just did it **2222539**, and this other is was created but rejected with a real credit card, payment id **2212162**

**Comment by scirielli** [comment]
Yes, my bad too. These are sandbox payments and right now you just can see them via api GET, you can't see them in the site's summary.

SC

**Comment by rflores00** [comment]
Ok, I got it.. But I'm trying to get payment info from this way `$paymentInfo = $mp->get_payment ("PAYMENT_ID");` and I got this result:

`{
   "status": false,
   "error":    {
      "classname": "MercadoPagoException",
      "message": "not_found"
   }
}`

Why is this happen?

Rubén,

**Comment by scirielli** [comment]
Ok, you should try:

```
curl -G -X GET \
-H "accept: application/json" \
"https://api.mercadopago.com/v1/payments/[ID]" \
-d "access_token=ACCESS_TOKEN"
```

with a test access token
Let me know if you're ok with that

SC

**Comment by rflores00** [comment]
I'm still getting an error, but this time is like a payment is not exist

{
   "message": "Payment not found",
   "error": "not_found",
   "status": 404,
   "cause": [   {
      "code": 2000,
      "description": "Payment not found",
      "data": null
   }]
}

**Comment by scirielli** [comment]
Yes, it is a problem with the sandbox api.
I'm labeling this issue as a bug, we'll try to fix it soon.

Regards,
SC

**Comment by scirielli** [comment]
Are you sure you're using Test credentials in the GET statement?
Could you send me the entire curl command you're using?

Thanks a lot,
SC
